cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -582,3 +583,134 @@ def test_quota_not_fails_after_vm_reconfigure_disk_remove(request, appliance, fu
 
         # Remove disk UI verification
         assert full_template_vm.configuration.num_disks == orig_config.num_disks
+
+
+@pytest.fixture(scope='module')
+def new_tenants(request, appliance):
+    """This fixture creates two parent tenants"""
+    tenants = []
+    for _ in range(2):
+        tenant = appliance.collections.tenants.create(
+            name=fauxfactory.gen_alphanumeric(18, start="test_parent_"),
+            description=fauxfactory.gen_alphanumeric(18, start="parent_desc_"),
+            parent=appliance.collections.tenants.get_root_tenant()
+        )
```

Comment:
Thoughts on using REST to create tenants?

Reply:
@valaparthvi 
- Tenants are very important with respect to **QUOTA** related test cases. So it is always better to create them **via UI**. So we can confirm that there is no issue while creating tenants, child tenants or projects and it's child.
- Hence I am keeping as it is for this as well as coming test cases of quota.